### ISSUE TITLE
Generating label for custom dataset

### ISSUE THREAD

**Issue by AleMaiaF** [zihangJiang/TokenLabeling#24]
Hello,

Thank you for sharing your work. I am currently trying to generate token label to a custom dataset for model lvvit_s, but I keep getting the loss close to 7 and the Accuracy 0 (not pre-trained and using 1 GPU in Google Colab). I also tried using the pre-trained model with --transfer  but got 0 in both Loss and Acc . What option should I use for a custom dataset?
![image](https://user-images.githubusercontent.com/51760453/152611422-c893a7de-3524-4f62-87f5-cc3a95e8044d.png)

**Comment by zihangJiang** [comment]
> Hello,
> 
> Thank you for sharing your work. I am currently trying to generate token label to a custom dataset for model lvvit_s, but I keep getting the loss close to 7 and the Accuracy 0 (not pre-trained and using 1 GPU in Google Colab). I also tried using the pre-trained model with --transfer but got 0 in both Loss and Acc . What option should I use for a custom dataset? ![image](https://user-images.githubusercontent.com/51760453/152611422-c893a7de-3524-4f62-87f5-cc3a95e8044d.png)

Are you using `generate_label.py` to generate label? 
Large loss and zero accuracy usually mean you are not using the correct number of classes for the label generation. My current implementation is only tested when class number = 1000. I'm not sure if `generate_label.py` works fine when you apply it on custom dataset. You may check with [this script](https://github.com/zihangJiang/TokenLabeling/blob/main/generate_label.py) and adapt it to your own dataset.

**Comment by veax-void** [comment]
>  with --transfer but got 0 in both Loss and Acc

It's because if transfer flag is used in [generate_label.py line 275](https://github.com/zihangJiang/TokenLabeling/blob/f291cbffdc48e37c248fc4a9ab1b7884c85c997f/generate_label.py#L275) disable tracking of loss and accuracy.  But it not clear to me why you can't collect them during transfer.
